static/templates/settings/account_settings.hbs

```diff
@@ -134,9 +134,21 @@
 
             <form class="deactivate_account grid">
                 <div class="input-group">
-                    <button type="submit" class="button rounded btn-danger" id="user_deactivate_account_button">
-                        {{t 'Deactivate account' }}
-                    </button>
+                    {{#if page_params.has_only_one_admin }}
```

Comment:
The two if conditions can be combined into a single condition `{{#if (and page_params.is_admin page_params.has_only_one_admin) }}`

Reply:
ok

Reply:
ok sure